main.sh

```diff
@@ -25,8 +25,11 @@ awk '/chr17/' hg19.coding_subset.sorted.bed | awk '$10==3' > hg19.coding_subset.
 awk 'BEGIN { OFS = "\t" } { $13 = $3 - $2 } 1' hg19.coding_subset.bed > hg19.coding_subset.length.bed
 
 ## Task6: starting with the original input file, we need to edit column 1 to remove the "chr" prefix in the chromosome name. For example, we will change chr17 to 17. Save the new file under the name "hg19.coding_subset.ensFormat.bed"
-
+sed 's/chr//' hg19.coding_subset.bed > hg19.coding_subset.ensFormat.bed
```

Comment:
This would remove the word "chr" anywhere in the file. We need something more careful